tests/test_utils.py

```diff
@@ -190,15 +190,16 @@ def test_can_undo_fp16_conversion(self):
         model = extract_model_from_parallel(model, keep_fp32_wrapper=False)
         _ = pickle.dumps(model)
 
-    @require_cuda
+    @require_triton
+    @require_non_cpu
```

Comment:
So by adding this, we enable this test on all non-CPU devices. Do we know that it passes on all of them? We don't cover non-CUDA on CI so we would not notice if we break it. Would it be better to explicitly check for XPU?

Reply:
I see your point, but this might be an opportunity to let them know that they should support it. And I think the previous device-agnostic work in transformers (#[25654](https://github.com/huggingface/transformers/issues/25654)) doesn't take into account whether the test case is supported yet or not. For devices that are not supported, they just add a manual skip. But I am not sure which way is the best to go for accelerate. Just let me know, I will update my code. :)